force-app/main/default/lwc/embeddedChatHandler/embeddedChatHandler.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @lwc/lwc/no-inner-html */

```

Comment:
Should we be disable the linter here?  If the innerHTML is just text which i believe it is in this case we should probably try and use the suggested salesforce suggestion and use `element.textContent` as described in the link below: 

https://github.com/salesforce/eslint-plugin-lwc/blob/master/docs/rules/no-inner-html.md

Reply:
Interesting, had not seen that, the innerHTML is used as part of a HTML decoding using a html node that is never added to the DOM actually, to decode the encoding Salesforce is doing when sending data from the chatbot to the lwc to handle messages.

Do you know if using inner text has the same behavior?

Reply:
I tried it and no it doesn't work with just passing text unfortunately:

```js
var txt = document.createElement("textarea");
txt.textContent = "We&#39;re unable to complete your request at this time.";
console.log(txt.childNodes[0].nodeValue);
//"We&#39;re unable to complete your request at this time."
```